MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -625,14 +625,18 @@ that the transaction took too long to commit.
 ``YDB_ERR_UNKNOWN`` — A call to `ydb_message()`_ specified an
 invalid message code.
 
-``YDB_ERR_VARNAMEINVALID`` — A  variable name is too long. [#]_
+``YDB_ERR_VARNAMEINVALID`` — A  variable name is invalid. [#]_
 
 .. [#] Note for implementers: While correctly issuing GVINVALID for
        too-long global variable names, YottaDB silently truncates
        local variable names that are too long. The implementation
        should catch this. ``YDB_ERR_VARNAMEINVALID`` can map to the
        existing GVINVALID, and change the message returned by
-       ``ydb_message()`` appropriately.
+       ``ydb_message()`` appropriately. Invalid variable
+       name errors are typically caught by the M compiler, resulting
+       in messages like EQUAL or RPARENMISSING depending on the
+       context. For the Simple API, all invalid variable names should
+       map to VARNAMEINVALID.
```

Comment:
Invalid subscripts too or just varnames (an invalid subscript is actually a missing subscript - i.e. a NULL parameter value)?

Reply:
As discussed in the last comment, let's go with a SUBSINVALID for invalid subscripts.

Reply:
As noted above, perhaps INVSUBSCRIPT?

Reply:
Keep it consistent with the variable name message, however that discussion comes out.